components/form/form-element-mixin.js

```diff
@@ -217,10 +222,7 @@ export const FormElementMixin = superclass => class extends LocalizeCoreElement(
 		return this.localize('components.form-element-mixin.valueMissingMessage', { label: this.labelText });
 	}
 
-	validationTooltipHide() { }
-
-	// eslint-disable-next-line no-unused-vars
-	validationTooltipShow(message) {}
+	validationTooltipShowHide() {}
```

Comment:
I think it makes sense to get rid of this altogether. I was thinking the component could do conditional rendering based on if `validationError` is non-null:
```
{this.validationError !== null ? html`<d2l-tooltip>${this.validationError}</d2l-tooltip>`} : null
```
The only risk there is it might make it a bit easier for people to forget to add tooltips to their custom form elements.